### DISCUSSION TITLE
Quickest way to initialise with "package mode = false"

### DISCUSSION THREAD

**Discussion by yinchi** [python-poetry/poetry#9447] [General] [answered]
I often use poetry for dependency management only and constantly need to edit the default pyproject.toml file to reflect this. Is there a quick way to initialise a project with no README.md, no [build-system], and with "package mode = false"?

Perhaps a new flag for "poetry init"?

**Comment by radoering** [comment, marked as answer]
There is no flag, but a PR introducing a new flag for `poetry init` and `poetry new` is welcome.

**Reply by yinchi** [reply]
I see. I've found src/poetry/console/commands but would need some time to understand how it works.

**Comment by ZackPlauche** [comment]
Please build this lmao

**Reply by airtonix** [reply]
needs more lolz

**Reply by ZackPlauche** [reply]
I mean I want a way to initialize poetry in a non-package mode as a starter command 🙂

**Comment by j2bbayle** [comment]
Agreed this would be super convenient!

**Comment by aarighi** [comment]
Is there a PR or an issue for this?

**Reply by j2bbayle** [reply]
Don't think there is…
